AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -439,7 +444,6 @@ - (void)checkLatestRelease:(NSString *)updateToken distributionGroupId:(NSString
                     // If the response payload is MSErrorDetails, consider it as a recoverable error.
                     if (!details || ![kMSErrorCodeNoReleasesForUser isEqualToString:details.code]) {
                       [MSKeychainUtil deleteStringForKey:kMSUpdateTokenKey];
-                      [MS_USER_DEFAULTS removeObjectForKey:kMSSDKHasLaunchedWithDistribute];
```

Comment:
Any reason of removing this line?

Reply:
The user will be re-prompted to sign in after an unrecoverable error. This was causing issues but i am not sure if we need that line to be removed now that we fixed the issue at another level.

Reply:
I can re-add it. It probably isn't needed anymore.